drivers/flash/flash_stm32_qspi.c

```diff
@@ -556,7 +556,7 @@ static void qspi_dma_callback(const struct device *dev, void *arg,
 {
 	DMA_HandleTypeDef *hdma = arg;
 
-	if (status != 0) {
+	if (status < 0) {
```

Comment:
also to be changed for ospi_dma_callback() function in the drivers/flash/flash_stm32_ospi.c  ?  ;-)

Reply:
The API change is done in a separate PR #52780

I updated it to include the change in the ospi driver, see https://github.com/zephyrproject-rtos/zephyr/pull/52780/commits/fc05656cb1fba72429513761d80970f0d3476a69

Reply:
Ok, I was confused  about the PR titlse :  there is more than use of dma circular buffer for the uart